machine/quanta/quanta_common_p2020/installer/install-platform

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# Overrides install.sh
+check_machine_image()
+{
+    platform=$(/bin/sed 's/quanta,//g' /proc/device-tree/compatible)
+    image_platform_machine=$(echo $image_machine | sed "s/common_p2020/${platform}/g")
+    echo "ONIE: Platform Machine: $image_platform_machine"
```

Comment:
Having read the entire patch I *think* I see what you are doing here.

Correct me if I'm wrong, but it looks like you have modified u-boot to "fix up" the device tree compatible and model names when booting OINE.  You do this by detecting the type of base board by reading a CPLD register?

If so that is kind of neat.  It is also new behavior that should be documented.  That documentation should go into the commit message and the INSTALL file.